packages/backend/src/controllers/users/dto/create-user.dto.ts

```diff
@@ -1,5 +1,8 @@
-import { ApiProperty } from '@nestjs/swagger';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 export class CreateUserDto {
+  @ApiPropertyOptional()
+  name: string;
```

Comment:
```suggestion
  name?: string;
```
it should instead be optional here